src/algebra/module/injective.lean

```diff
@@ -325,7 +325,7 @@ def extension_of_max_adjoin (h : module.Baer R Q) (y : N) :
           ↑((extension_of_max_adjoin.fst i a) + (extension_of_max_adjoin.fst i b)) +
           (extension_of_max_adjoin.snd i a + extension_of_max_adjoin.snd i b) • y,
         { rw [extension_of_max_adjoin.eqn, extension_of_max_adjoin.eqn, add_smul],
-          abel, },
+          abel_nf, },
```

Comment:
Why is this abel_nf if it closes the goal?

Reply:
there are a few like this, in some of the calc blocks also

Reply:
I see that this is becaus `abel1` fails, which I feel like it shouldnt here, is this an issue you know of?

Reply:
`abel_nf` is slightly more powerful than `abel1` even on equations, because it will normalize inside atoms recursively using `abel_nf`. Not sure whether that is what is happening here, but it is not impossible for a goal to be closed by `abel_nf` but not `abel1`. (I consider it a bit smelly though.)